### DISCUSSION TITLE
Add collection name to the tab title

### DISCUSSION THREAD

**Discussion by w108x** [gotson/komga#1710] [Ideas]
Right now the tab title in the browser only shows "Komga" for tabs that have a collection open, it would be more useful to show the collection name as well. And when on the main view of a library (not in a series or collection) I guess displaying the library name instead of just "Komga" as well.

**Comment by gotson** [comment]
I've added this for Dashboard, and when browsing libraries, collections and readlists

**Comment by gotson** [comment]
🎉 This issue has been resolved in `1.19.0` ([Release Notes](https://github.com/gotson/komga/releases/tag/1.19.0))
